screens/ModeratorHomeScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import { ScrollView, StyleSheet } from 'react-native';
+import Notification from '../components/Notification';
+import backendFetch from '../lib/backendFetch';
+import React, { useEffect, useState } from 'react';
+import User from '../types/User';
+import Loading from '../components/Loading';
+
+export default function ModeratorHomeScreen() {
+  const [companies, setCompanies] = useState<User[]>();
+  useEffect(() => {
+    backendFetch<User[]>('GET', 'company')
+      .then(companies => setCompanies(companies as User[]))
```

Comment:
Again, I recommend extracting to a different function. This would mean you can just do `getCompanies().then(setCompanies)`